csharp/Facebook.Yoga/YogaNode.Spacing.cs

```diff
@@ -17,7 +15,7 @@ public YogaValue Left
         {
             get
             {
-                return Native.YGNodeStyleGetPosition(_ygNode, YogaEdge.Left);
+                return StructHelpers.MarshalStruct<YogaValue>(Native.YGNodeStyleGetPosition(_ygNode, YogaEdge.Left));
```

Comment:
Humm i not very fan of this.. using generics will have a performance hit on Android.